portal/templates/student_views/schedule.html

```diff
@@ -10,7 +10,7 @@ <h2>Schedule for {{ g.user['name'] }}</h2>
   <a href="{{ url_for('index') }}"  class='back'>Back</a>
 
 </header>
-
+<section id='hover' class='student_grades'>
   <table class="schedule">
```

Comment:
Not a huge deal, but it seemed weird to me to have single quotes for one class, but double quotes for another one